src/celeritas/optical/OpticalPropertyData.hh

```diff
@@ -15,6 +15,21 @@
 
 namespace celeritas
 {
+//---------------------------------------------------------------------------//
+/*!
+ * Material dependent scintillation spectra.
+ *
+ * Support upto 3-components: [fast, medium, slow]
+ */
+struct ScintillationSpectra
+{
```

Comment:
Should we have an enumeration for this? Should it be a runtime selectable value? Since the 3 components aren't spatial x/y/z, perhaps we should do:
```suggestion
{
    static inline constexpr size_type size = 3;
    using Real3 = Array<real_type, size>;
```